index.js

```diff
@@ -1,20 +1,44 @@
-let { readFile, stat } = require("fs").promises;
-let path = require("path");
-let { FileFinder } = require("faucet-pipeline-core/lib/util/files/finder");
+let { readFile, stat } = require("node:fs/promises");
+let path = require("node:path");
+let { FileFinder } = require("./lib.js");
```

Comment:
IMO root-level `index.js` is only acceptable if there's exactly one source file, so you'll want to introduce a `lib` directory here? (This would also be true if it was just `types.ts`.)

Also, as a personal favor, can you move the local import to the top? ( 🎗️ local > third party > standard library)

Reply:
both done in 341098f4ff4c62194a7e575828012112cb804b39